client/src/utils/saveAsPDF.ts

```diff
@@ -0,0 +1,24 @@
+import { Election } from '@votingworks/ballot-encoder'
+
+export default async function saveAsPDF(
+  fileNamePrefix: string,
+  election: Election,
+  precinctName?: string
+): Promise<void> {
+  const precinctNameInFileName = precinctName || 'all-precincts'
```

Comment:
This would be better done with a default parameter value:

```suggestion
  precinctName = 'all-precincts'
): Promise<void> {
```